src/services/db/RepoService.ts

```diff
@@ -113,9 +113,36 @@ export default class RepoService extends GitHubService {
   }
 
   async create(
-    sessionData: any,
-    { content, fileName, directoryName, isMedia = false }: any
-  ): Promise<any> {
+    sessionData: UserWithSiteSessionData,
+    {
+      content,
+      fileName,
+      directoryName,
+      isMedia = false,
+    }: {
+      content: string
+      fileName: string
```

Comment:
will filename be there for create dir too? if not, is this optional? 
some thinking points - would it be better to have 2 separate functions for createFile createDir or maybe a flag like isDir to make it explicit?

Reply:
Yup it'll be there as well, create dir operations are just creating a `.keep` file in the desired directory currently